app/containers/message/Components/RightIcons/Translated.tsx

```diff
@@ -0,0 +1,23 @@
+import React, { memo } from 'react';
+import { View } from 'react-native';
+
+import { CustomIcon } from '../../../CustomIcon';
+import { useTheme } from '../../../../theme';
+import { themes } from '../../../../lib/constants';
+import styles from '../../styles';
+
+const Translated = memo(({ isTranslated }: { isTranslated: boolean }) => {
```

Comment:
You need to add this component to Storybook

<img width="502" alt="image" src="https://github.com/RocketChat/Rocket.Chat.ReactNative/assets/804994/fcaa0606-4c66-4cac-9092-129ee4ca0436">